docs/infra/standaarden/haven/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Haven (Kubernetes)"
+sidebar_position: 0
+tags:
+    - "kubernetes"
+    - "infra"
+    - "containerization"
+    - "helm"
+    - "docker"
+---
+
+Haven is een standaard voor platform-onafhankelijke cloud hosting. Het is een project dat is ontstaan vanuit de wens van Nederlandse gemeenten om op een uniforme manier hun Kubernetes cloud hosting te organiseren. Het project is onderdeel van de Common Ground architectuurvisie. Common Ground is een initiatief van de Vereniging van Nederlandse Gemeenten (VNG) waarin samengewerkt wordt aan principes voor een toekomstbestendige digitale architectuur.
+
+![Haven](./img/haven_overview.svg)
+
+## Voordelen
```

Comment:
Ik zou hierboven een paragraaf invoegen: "De standaard", of iets dergelijks, waar we een duidelijke verwijziing maken naar de specificatie van de standaard: https://haven.commonground.nl/techniek/checks
En misschien kunnen we daar meteen een aantal voorbeelden noemen van checks, waardoor de lezer wat meer gevoel krijgt bij de inhoud van de standaard.